watchtower/blob/justice_kit.go

```diff
@@ -0,0 +1,406 @@
+package blob
+
+import (
+	"bytes"
+	"encoding/binary"
+	"errors"
+	"fmt"
+	"io"
+
+	"golang.org/x/crypto/chacha20poly1305"
+
+	"github.com/btcsuite/btcd/btcec"
+	"github.com/lightningnetwork/lnd/lnwallet"
+	"github.com/lightningnetwork/lnd/lnwire"
+)
+
+const (
+	// MinVersion is the minimum blob version supported by this package.
+	MinVersion = 0
+
+	// MaxVersion is the maximumm blob version supported by this package.
+	MaxVersion = 0
+
+	// NonceSize is the length of a chacha20poly1305 nonce, 12 bytes.
+	NonceSize = chacha20poly1305.NonceSize
+
+	// KeySize is the length of a chacha20poly1305 key, 32 bytes.
+	KeySize = chacha20poly1305.KeySize
+
+	// CiphertextExpansion is the number of bytes padded to a plaintext
+	// encrypted with chacha20poly1305, which comes from a 16-byte MAC.
+	CiphertextExpansion = 16
+
+	// V0PlaintextSize is the plaintext size of a version 0 encoded blob.
```

Comment:
Could probably copy over the comment in the body of the PR breaking down the side to this `godoc` string.

Reply:
Done